mail_original_from/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Mail Original From
+==================
+
```

Comment:
Should not we rather put this section under usage?
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst